open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,24 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
```

Comment:
Are query parameters case sensitive or insensitive? 

Can I make a request to `v1/prefix/namespaces?pagetoken=uuid&pagesize=500&parent=accounting` and expect the same behavior?

Reply:
Looks like OpenAPI query parameter definition is case sensitive, which I think is fine given we already have `purgeRequested`.